src/main/java/uk/gov/pay/api/utils/TrustStoreLoader.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.api.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+public class TrustStoreLoader {
```

Comment:
CERTS_PATH might be a mounted docker volume, so removing the certs might be a bad idea. I don't really follow your point though, surely a restart of the DropWizard app is exactly when we want to check for cert changes. The TrustStore is new every time, so there's no chance of overlapping or adding the same cert twice